wunderio/next4drupal-project
# NEX-42: Override the ckeditor5 image resize plugin to use pixels instead of percentages

*Link to ticket: https://wunder.atlassian.net/browse/NEX-42 *

This PR implements a hook to change the configuration of the core-provided `ImageResize` ckeditor plugin to resize images by px instead of percentages.

### To test

1. lando drush cr
2. add a picture in a wysiwyg field, resize the image, you will see that it's resized in pixels
![image](https://user-images.githubusercontent.com/185412/218705491-7bde0dea-0694-4775-a489-f1484ad7467f.png)
3. the image should appear correctly in the frontend.
